content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -75,8 +75,15 @@ The following bulk operations run as transactions:
 
 The `$transaction` API can be used in two ways:
 
-- `$transaction<R>(queries: PrismaPromise<R>[]): Promise<R[]>`: Pass an array of Prisma Client queries to be executed sequentially inside of a transaction.
-- `$transaction<R>(fn: (prisma: PrismaClient) => R): R`: Pass a function that can contain user code including Prisma Client queries, non-Prisma code and other control flow to be executed in a transaction.
+- **Sequential interactions**: Pass an array of Prisma Client queries to be executed sequentially inside of a transaction.
+
+  `$transaction<R>(queries: PrismaPromise<R>[]): Promise<R[]>`
+  
+- **Interactive transactions (Preview)**: Pass a function that can contain user code including Prisma Client queries, non-Prisma code and other control flow to be executed in a transaction.
+
+  `$transaction<R>(fn: (prisma: PrismaClient) => R): R`
+
+> **Note**: to use interactive transactions you will need to [enable it as a Preview feature](/guides/upgrade-guides/upgrading-to-use-preview-features#enabling-preview-features).
```

Comment:
Should this link maybe just go down to the "Interactive Transactions (in Preview)" headline as that section starts with info about the preview feature?

Reply:
It could, but the section below didn't actually explain how to do it (only shows code example, true)... I'd like us to start being very consistent and always link to the page that explains how to enable any Preview feature.